foreman.fc

```diff
@@ -34,14 +34,19 @@
 # Passenger non-SCL file contexts
 
 /usr/lib/ruby/gems/1.8/gems/passenger-.*/agents/.* -- gen_context(system_u:object_r:passenger_exec_t,s0)
-/usr/lib64/ruby/site_ruby/1.8/x86_64-linux/agents/.* -- gen_context(system_u:object_r:passenger_exec_t,s0)
+/usr/lib/ruby/site_ruby/1.8/x86_64-linux/agents/.* -- gen_context(system_u:object_r:passenger_exec_t,s0)
```

Comment:
I think this needs to stay as `lib64`, as EL6 doesn't seem to have the lib/lib64 equivalence rules. This rule isn't used on EL7, it's an EL6-only path.

<pre>
# ll -Z /usr/lib64/ruby/site_ruby/1.8/x86_64-linux/agents
-rwxr-xr-x. root root system_u:object_r:lib_t:s0       PassengerHelperAgent
-rwxr-xr-x. root root system_u:object_r:lib_t:s0       PassengerLoggingAgent
-rwxr-xr-x. root root system_u:object_r:lib_t:s0       PassengerWatchdog
-rwxr-xr-x. root root system_u:object_r:lib_t:s0       SpawnPreparer
# restorecon -Rvv /usr/lib64/ruby/site_ruby/1.8/x86_64-linux/agents
[no output]
</pre>